plugins/factory-plugin/src/theia-commands.ts

```diff
@@ -28,7 +29,33 @@ function isDevfileProjectConfig(project: cheApi.workspace.ProjectConfig | cheApi
     return !!project.name && !!project.source && !!project.source.type && !!project.source.location && !project.source['parameters'];
 }
 
-export class TheiaCloneCommand {
+export interface TheiaCloneCommand {
+    execute(): PromiseLike<void>;
+}
+
+export function getTheiaCloneCommand(
```

Comment:
maybe rename it `getProjectImportCommand` or `createProjectImportCommand` or `buildProjectImportCommand` or `generateProjectImportCommand`
but zip is not about cloning so we should avoid using the word clone imho

Reply:
thanks, Sun
`buildProjectImportCommand` sounds good to me